src/v3/test/integration/__snapshots__/authenticator-enroll-phone-sms.test.tsx.snap

```diff
@@ -109,9 +110,9 @@ exports[`authenticator-enroll-phone-sms should render form 1`] = `
                 >
                   <label
                     class="MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-root MuiFormLabel-colorPrimary emotion-14"
-                    for="generated"
+                    for="credentials.passcode"
                   >
-                    Enter code
+                    Enter Code
```

Comment:
incorrect key mapping?

Reply:
This looks to be correct, this is directly from v2 i18n transformer override map: https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/i18nTransformer.js#L108